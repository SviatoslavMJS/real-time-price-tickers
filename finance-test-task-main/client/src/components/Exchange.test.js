import renderer from 'react-test-renderer';
import Exchange from './Exchange.jsx';

test("On change prop exchange Exchange text must change from 666 to 000", () => {
    const component = renderer.create(
      <Exchange exchange ={"666"} />,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();

    tree.props.exchange = "000";

    tree = component.toJSON();
    expect(tree).toMatchSnapshot();

    tree.props.exchange = "666";

    tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  
});