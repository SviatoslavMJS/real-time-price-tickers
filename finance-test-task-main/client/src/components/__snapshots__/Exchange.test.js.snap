// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`On change prop exchange Exchange text must change from 666 to 000 1`] = `
<div
  className="exchange"
>
  666
</div>
`;

exports[`On change prop exchange Exchange text must change from 666 to 000 2`] = `
<div
  className="exchange"
>
  666
</div>
`;

exports[`On change prop exchange Exchange text must change from 666 to 000 3`] = `
<div
  className="exchange"
>
  666
</div>
`;
